packages/calcite-components/src/utils/openCloseComponent.ts

```diff
@@ -1,10 +1,9 @@
-import { readTask } from "@stencil/core";
 import { whenTransitionDone } from "./dom";
 
 /**
  * Exported for testing purposes only
  */
-export const internalReadTask = readTask;
+export const internalRaf = requestAnimationFrame;
```

Comment:
I'll try to remove this before installing.

Reply:
Yeah seems like this just needs to be cleaned up to just use requestAnimationFrame by itself.